docs/APPARMOR.md

```diff
@@ -0,0 +1,21 @@
+# apparmor
+
+When running Telegraf under apparmor users may see denial messages depending on
```

Comment:
s/apparmor/AppArmor/ here and everywhere it is listed as a product name (ie, not in a denial message)